blog/index.html

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
To be consistent with the other pages just keep this in .md file. I know pretty much all of this is html but the goal in general is to write as much in markdown as possible unless you're working with a template. These index pages are exceptions but should still ideally be as much markdown as possible.